app/main/db/queries.py

```diff
@@ -256,14 +256,14 @@ def _build_series_view_query(series_id, filters, sorting_orders):
 def _build_browse_filters(query, sub_query, filters):
     transferring_body = filters.get("transferring_body")
     if transferring_body:
-        filter_value = str(f"{transferring_body}%").lower()
+        filter_value = str(f"%{transferring_body}%").lower()
```

Comment:
so I assume %STRING% is the format for a CONTAINS string filter in sqlalchmey

Reply:
if you want to match characters that are contained in a given string, you surround the string with percentage signs (%) on both sides of the filter value string. i.e. %t% will get results for test, star, dest, into and so on

Reply:
perfect, thanks for explaining 👌